### PR TITLE
New rules for processing AddCommand for providing vrHelp items and title

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Added information about limitation for addCommands and processed addCommand type according to https://github.com/smartdevicelink/sdl_evolution/issues/364.

### CODE REVIEW

**Comment by theresalech** [comment]
@GetmanetsIrina can you please advise if these revisions are ready for review?

**Comment by GetmanetsIrina** [comment]
@theresalech , these revisions are ready for review. 
Unfortunately have no permissions to add reviewers, labels and so on.

File: proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The proposed mechanism is detailed below:
   	2. construct the `helpPrompt` parameter using the data from the list SDL Core internally created.
   	3. then send these parameters to the HMI via the `SetGlobalProperties` RPC.
 5. If after SDL Core sends the `SetGlobalProperties` RPC, applications sends further AddCommmand/DeleteCommand requests then SDL Core shall send `SetGlobalProperties` with the updated parameters.(The full list of parameters needs to be sent and not just the recently added one).
+The limitation for processing AddCommand and providing updated values of "vrHelp" and "helpPrompt" to HMI till mobile application sends SetGlobalProperties request is 30.
```
Comment (theresalech) [inline proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md]: Please update "...till mobile application..." to "...until the mobile application sends..."

File: proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ The proposed mechanism is detailed below:
...
+Update values of "vrHelp" and "helpPrompt" via TTS UI.SetGlobalProperties to HMI only by receving AddCoomad with CommandType = Command. AddCommand requests related to choice set must not trigger the update of "vrHelp" and "helpPrompt" values.
```
Comment (theresalech) [inline proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md]: Please fix the following typos:
- "receving" should be "receiving"
- "AddCoomad" should be "AddCommand"

**Comment by GetmanetsIrina** [comment]
@theresalech, please find update 6b9125a104844bf1368b88821a89c0a47782009e.

File: proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md [hunk 3]
```diff
@@ -26,6 +26,8 @@ The proposed mechanism is detailed below:
   	2. construct the `helpPrompt` parameter using the data from the list SDL Core internally created.
   	3. then send these parameters to the HMI via the `SetGlobalProperties` RPC.
 5. If after SDL Core sends the `SetGlobalProperties` RPC, applications sends further AddCommmand/DeleteCommand requests then SDL Core shall send `SetGlobalProperties` with the updated parameters.(The full list of parameters needs to be sent and not just the recently added one).
+The limitation for processing AddCommand and providing updated values of "vrHelp" and "helpPrompt" to HMI until the mobile application sends SetGlobalProperties request is 30.
+Update values of "vrHelp" and "helpPrompt" via TTS UI.SetGlobalProperties to HMI only by receiving AddCommad with CommandType = Command. AddCommand requests related to choice set must not trigger the update of "vrHelp" and "helpPrompt" values.
```
Comment (theresalech) [inline proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md]: "AddCommad" should be "AddCommand"

**Comment by GetmanetsIrina** [comment]
@theresalech , please find update in 146d7e6b634ca97fceab38a1d826dd920525b74a.

**Pull request merged**